README.md

```diff
@@ -105,25 +105,53 @@ java -jar fcrepo-import-export.jar --mode import --resource http://example.org:8
+The import-export-utility supports import and export of [BagIt](https://tools.ietf.org/html/draft-kunze-bagit-14) bags 
+and has BagIt specific command line arguments in order to support a number of use cases.
+
+##### Bag Profile
 
 To enable a bag profile, use the `-g` or `--bag-profile` option. The import/export utility currently supports the following bag profiles:
 
-* [default](src/main/resources/profiles/default.json)
-* [aptrust](src/main/resources/profiles/aptrust.json)
-* [metaarchive](src/main/resources/profiles/metaarchive.json)
-* [perseids](src/main/resources/profiles/perseids.json)
-* [beyondtherepository](src/main/resources/profiles/beyondtherepository.json)
+* [default](https://raw.githubusercontent.com/duraspace/bagit-support/master/src/main/resources/profiles/default.json)
+* [aptrust](https://raw.githubusercontent.com/duraspace/bagit-support/master/src/main/resources/profiles/aptrust.json)
+* [metaarchive](https://raw.githubusercontent.com/duraspace/bagit-support/master/src/main/resources/profiles/metaarchive.json)
+* [perseids](https://raw.githubusercontent.com/duraspace/bagit-support/master/src/main/resources/profiles/perseids.json)
+* [beyondtherepository](https://raw.githubusercontent.com/duraspace/bagit-support/master/src/main/resources/profiles/beyondtherepository.json)
 
+##### Bag Metadata
 
-For example, to export all of the resources from a Fedora repository at `http://localhost:8080/rest/` in a BagIt bag using the default profile and user supplied metadata for tag files:
+You can export a [BagIt](https://tools.ietf.org/html/draft-kunze-bagit-14) bag from a Fedora repository based on a 
+[BagIt Profile](https://github.com/ruebot/bagit-profiles/). User supplied metadata for tag files can be provided with a 
+Yaml file specified by the `-G` or `--bag-config` option.
 
-```sh
-java -jar fcrepo-import-export.jar --mode export --resource http://localhost:8080/rest --dir /tmp/example_bag --binaries --bag-profile default --bag-config /tmp/bagit-config.yml
+The configuration file specified must have a top-level key matching the name of the metadata file with sub keys for
+each field you wish to manually supply. For example, setting metadata elements in the `bag-info.txt`:
+
+```yaml
+bag-info.txt:
+  Source-Organization: org.fcrepo
+  Organization-Address: https://github.com/fcrepo4-labs/fcrepo-import-export
 ```
 
-Where `bagit-config.yml` looks like:
+##### Bag Serialization
+
+The import-export-utility supports serialization as part of import and export. 
+
+During import the utility will attempt to extract the serialized bag based on the content type of the path specified.
+
+For export, if a bag profile allows serialization the format can be specified with `-s` or `--bag-serialization` along 
+with the desired format. Currently, the following formats are supported:
+* `tar`
```

Comment:
It appears that only `tar` is supported.

Reply:
I'll double check this. I suspect because the format depends on the bag profile being used we probably want to give feedback if there's is a conflict between what the user wants and what the profile allows.

Reply:
I added `Profile Requirements` to address this after talking with Bill. We figured since each profile will have different formats which are allowed, it didn't make sense to simply list all the formats which are available. So instead there's a table which lists the allowed formats for each profile.